### PR TITLE
Editor overhaul / ARIA improvements

### LINKED ISSUE #2841: Clean up the core editor modules
The editor in use pretty much has reached its end of life, with updates coming very rarely and only taking care of minor issues, that we usually fix ourselves far more in advance. Regardless, it has been become very stable and mature over the course of time and while there are a few reasons why we cannot make a switch to a different editor at this time, we can still make the best out of it.

### PR DESCRIPTION
**Pull request by dtdesign**
See #2713 
Closes #2841

### CODE REVIEW

File: wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js [hunk 1]
```diff
@@ -78,24 +31,12 @@ $.Redactor.prototype.WoltLabDropdown = function() {
 		},
 		
 		_show: function() {
+			return;
```
Comment (TimWolla) [inline wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js]: Is this an intentional change?

File: wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Dropdown/Simple.js [hunk 2]
```diff
@@ -513,62 +519,83 @@ define(
...
+					button = dropdown;
+				}
+				
 				this._toggle(null, _activeTargetId);
 				if (button) button.focus();
 			}
+		},
+		
+		_getNextVisibleItem: function(activeItem, index) {
+			var items = Array.prototype.slice.call(elBySelAll('li', activeItem.closest('.dropdownMenu')));
```
Comment (TimWolla) [inline wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Dropdown/Simple.js]: This method does not look correct to me.

File: wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js (hunk 1 above)
Comment (dtdesign) [reply wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js]: Yes, this was during development and a later commit removed this file entirely.

File: wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Dropdown/Simple.js (hunk 2 above)
Comment (dtdesign) [reply wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Dropdown/Simple.js]: It's a leftover method, I'll remove it, thanks.

File: wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js (hunk 1 above)
Comment (TimWolla) [reply wcfsetup/install/files/js/3rdParty/redactor2/plugins/WoltLabDropdown.js]: I believe I already was reviewing before that commit appeared :smile:

**Pull request merged**